lib/phare/cli.rb

```diff
@@ -43,12 +41,17 @@ def symbolize_options!(options)
       options
     end
 
+    # rubocop:disable Metrics/AbcSize
     def parsed_options_from_arguments(argv)
       options_to_merge = {}
 
       OptionParser.new do |opts|
         opts.banner = 'Usage: phare [options]'
 
+        opts.on('--version', 'The version of Phare') do
```

Comment:
It should be `'Display Phare’s version'`. This is not like `--directory` where we indicate what the option *takes* as an argument :smile: